src/test/java/com/mathworks/ci/MatlabBuilderTest.java

```diff
@@ -279,4 +282,134 @@ public void verifyCustomCommandInvokedForBatchMode() throws Exception {
         jenkins.assertLogContains("-batch", build);
         jenkins.assertLogContains("runtests", build);
     }
+    
+    /*
+     * Test to verify if Automatic option passes appropriate test atrtifact values.
+     */
+
+    @Test
+    public void verifyRunTestAutomaticallyIsDefault() throws Exception {
+        this.matlabBuilder.setLocalMatlab(getMatlabroot("R2018b"));
+        RunTestsAutomaticallyOption runOption = new RunTestsAutomaticallyOption();
+        runOption.setTaCoberturaChkBx(true);
+        runOption.setTaJunitChkBx(true);
+        runOption.setTatapChkBx(true);
+        this.matlabBuilder.setTestRunTypeList(runOption);
+        project.getBuildersList().add(this.matlabBuilder);
+        FreeStyleBuild build = project.scheduleBuild2(0).get();
+        jenkins.assertLogContains("-batch", build);
+        jenkins.assertLogContains("true,true,true", build);
```

Comment:
Is there any chance this test could pick up the wrong output?  Is there some way to make it clearer that these boolean values came from the checkboxes?

Reply:
these have chance of taking default values if someone changes the .jelly file with corresponding web element names , but forgets to add it in action class. i don't see direct way to map these to confirm if its coming from UI other than writing some Web automation (functional tests) . I got your point though.